ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -888,7 +900,6 @@ public FindInfo findEntriesSince(FindToken token, long maxTotalSizeOfEntries, St
         remoteTokenTracker.updateTokenFromPeerReplica(token, hostname, remoteReplicaPath);
       }
       FindInfo findInfo = index.findEntriesSince(token, maxTotalSizeOfEntries);
-      onSuccess();
```

Comment:
findEntriesSince doesn't perform any disk io since all the index files are already loaded in the memory.